app/routes/groups_.$groupId.tsx

```diff
@@ -3,20 +3,146 @@ import { useParams, useLoaderData, Link } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
 import { json } from '@remix-run/node';
 
+import { Button } from '~/components/ui/button';
+
+import { Image } from '~/components/ui/images';
+
 export const loader: LoaderFunction = async ({ params }) => {
   const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
   return json({ group });
 };
 
 export default function GroupRoute() {
   const data = useLoaderData<typeof loader>();
-
   return (
-    <>
-      <Link to="/groups">back to groups</Link>
-      <h1>{data.group.name}</h1>
-      <p>{data.group.description}</p>
-    </>
+    <div className="bg-primary text-white flex flex-col items-center">
+      <div className="md:flex md:flex-col md:max-w-screen-xl">
+        <div className="p-2 justify-center md:p-10 md:flex md:flex-row md:border-b-2 md:border-b-white">
+          <div className="m-auto rounded-full h-56 w-52  md:h-80 md:w-72 justify-center items-center md:m-8">
+            {!data.group.imgUrl && !data.group.imgAlt ? (
+              <Image
+                className="w-full"
+                src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708118888/default-group-photo_xhcpqt.png"
```

Comment:
More a question for a Monday session, is there a preference or a point when long URLs, etc get stored in a constants file? I'd imagine a variable like this could be imported.

```javascript
const BUDDIES_SELFIE_IMG_URL = "https://res.cloudinary.com/dxctpvd8v/image/upload/v1708118888/default-group-photo_xhcpqt.png"

src={BUDDIES_SELFIE_IMG_URL}
```